pkgs/by-name/st/stalwart-mail/package.nix

```diff
@@ -25,13 +27,11 @@ rustPlatform.buildRustPackage {
     owner = "stalwartlabs";
     repo = "mail-server";
     rev = "v${version}";
-    hash = "sha256-OHwUWSUW6ovLQTxnuUrolQGhxbhp4YqKSH+ZTpe2WXc=";
+    hash = "sha256-Hpb7/GLrbZkruY3UTWdwIzwiwgcCT/JzFnUH5tCZaOQ=";
     fetchSubmodules = true;
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-  };
+  cargoHash = "sha256-/q+27KM/syWmRUiXhrzRqG8arjD007jL5JedU4RGC20=";
 
```

Comment:
Stalwart ships some interesting stuff like spamfilter config and a webinterface, that we could include into the derivation.
Possibly we could gate that with arguments, if you think tht is not generally needed.

i am doing it in my personal derivation (that i am testing atm) with the following snippet

```nix
  postInstall = ''
    mkdir -p $out/etc/stalwart $out/share/web/
    cp resources/config/spamfilter.toml $out/etc/stalwart/spamfilter.toml
    cp -r resources/config/spamfilter $out/etc/stalwart/
    cp resources/webadmin.zip $out/share/web/
  '';
```

Reply:
This is a really good addition, but in case this is optional, I would like to have this into a separate PR because this is already quite complex